src/schema/artwork/index.js

```diff
@@ -122,6 +122,88 @@ export const artworkFields = () => {
     category: {
       type: GraphQLString,
     },
+    attribution_class: {
+      type: GraphQLString,
+      description: "Attribution class as classification name",
+      resolve: artwork => {
```

Comment:
#minor but the signature can be written as `resolve: ({ attribution_class }) => {` and you can drop the `artwork.` part in the function. Up to you!